### PR TITLE
fix: sandbox-break-out-nunjucks-template-engine

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

http://disse.cting.org/2016/08/02/2016-08-02-sandbox-break-out-nunjucks-template-engine

### CODE REVIEW

File: test/view/security.test.js [hunk 1]
```diff
@@ -91,4 +91,170 @@ describe('test/view/security.test.js', () => {
     assert($('#script2').attr('nonce') === expectedNonce);
     assert($('#script3').attr('nonce') === expectedNonce);
   });
+
+  // http://disse.cting.org/2016/08/02/2016-08-02-sandbox-break-out-nunjucks-template-engine
+  describe('sandbox-break-out-nunjucks-template-engine', () => {
+    it('allow {{7*7}}', () => {
+      return app.httpRequest()
+        .get('/sandbox')
+        .query({ tpl: '{{7*7}}' })
```
Comment (atian25) [inline test/view/security.test.js]: 这种方式还是没禁止

Comment (fengmk2) [reply test/view/security.test.js]: 这种如何禁止？

**Comment by fengmk2** [comment]
https://github.com/node-modules/nunjucks/pull/5

**Comment by fengmk2** [comment]
2.1.4

File: test/view/security.test.js (hunk 1 above)
Comment (atian25) [reply test/view/security.test.js]: 嗯，没得禁止，本来就是意图

**Pull request merged**